src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -48,6 +48,10 @@
   <module name="org.openhab.tools.analysis.checkstyle.OverridingParentPomConfigurationCheck">
     <property name="severity" value="warning" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.RequireBundleCheck">
+      <property name="severity" value="warning" />
```

Comment:
I changed the severity to "error". There are 6 bundles that break the build, because their `MANIFEST.MF` files contain `Require-Bundle` header: 
```
[INFO] Freebox Binding .................................... FAILURE [ 10.428 s]
[INFO] MAX! Binding Tests ................................. FAILURE [ 24.919 s]
[INFO] Network Binding .................................... FAILURE [  9.790 s]
[INFO] Rfxcom Binding Tests ............................... FAILURE [ 10.066 s]
[INFO] Vitotronic Binding ................................. FAILURE [  8.072 s]
[INFO] PHP Support for the CometVisu Backend .............. FAILURE [  8.815 s]
```
Do you think that any of them should be added to `suppressions.xml` file?